en/ha.rst

```diff
@@ -1808,7 +1808,7 @@ Load Balancing Structure
 
 The load balancing structure increases the availability of the CUBRID service by placing several nodes in the HA configuration (one master node and one slave node) and distributes read-load.
 
-Because the replica nodes receive replication logs from the nodes in the HA configuration and maintain the same data, and because the nodes in the HA configuration do not receive replication logs from the replica nodes, its network and disk usage rate is lower than that of the multiple-slave structure.
+Because the replica nodes receive replication logs from whole nodes in the HA configuration and maintain the same data, and because whole nodes in the HA configuration do not receive replication logs from the replica nodes, its network and disk usage rate is lower than that of the multiple-slave structure.
```

Comment:
This part seems to be a limited description of 'Load Balancing Structure'. In order to explain the blue dotted line of 'An Sample of Node Configuration' figure, 'the master node' seems to be the correct modification.

Reply:
OK, I got it. I will change it with your suggestion.